src/cli/task-telemetry.ts

```diff
@@ -0,0 +1,42 @@
+import { getCompilerSystem, getLogger, getStencilCLIConfig } from './state/stencil-cli-config';
+import { checkTelemetry, disableTelemetry, enableTelemetry } from './telemetry/telemetry';
+
+export const taskTelemetry = async () => {
+  const logger = getLogger();
+  const prompt = logger.dim(getCompilerSystem().details.platform === 'windows' ? '>' : '$');
+  const isEnabling = getStencilCLIConfig().flags.args.includes('on');
+  const isDisabling = getStencilCLIConfig().flags.args.includes('off');
+  const INFORMATION = `Opt in or our of telemetry. Information about the data we collect is available on our website: ${logger.bold(
+    'https://stenciljs.com/telemetry',
+  )}`;
+  const THANK_YOU = `Thank you for helping to make Stencil better! 💖`;
+  const ENABLED_MESSAGE = `${logger.green('Enabled')}. ${THANK_YOU}\n\n`;
+  const DISABLED_MESSAGE = `${logger.red('Disabled')}\n`;
+  let hasTelemetry = await checkTelemetry();
```

Comment:
This variable could be `const`, as we never reassign it
```suggestion
  const hasTelemetry = await checkTelemetry();
```